src/function/localStorage.test.js

```diff
@@ -0,0 +1,54 @@
+import StoreLists from './localStorage.js';
+import TestList from './testList.js';
+
+describe('should check the item is added', () => {
+  test('should add to the list', () => {
+    const test = new TestList(1, 1, 'sleep', false);
+    const testlist = StoreLists.add(test);
+    expect(testlist).toHaveLength(1);
+  });
+
+  test('should add to the list', () => {
+    const test = new TestList(2, 2, 'finish the project', false);
+    const testlist = StoreLists.add(test);
+    expect(testlist).toHaveLength(2);
+  });
+});
+
+describe('should edit to the input', () => {
+  test('should edit the input', () => {
+    const test = new TestList(3, 3, 'test', false);
+    StoreLists.addList(test);
+    StoreLists.editList(3, 'meet');
+    expect(JSON.parse(localStorage.getItem('todoList'))[0].name).toBe('meet');
+  });
+});
+
+describe('update task completed status', () => {
+  test('test should change completed  status', () => {
+    const testItem = new TestList(1, 1, 'drink', false);
+    StoreLists.addList(testItem);
+    const array = StoreLists.getList();
+    StoreLists.completeLists(array, true, array[0].id);
+    expect(JSON.parse(localStorage.getItem('todoList'))[0].completed).toBeTruthy();
+  });
+});
+
+describe('Deleteing task that are complete', () => {
+  test('test for delete checked function', () => {
+    localStorage.clear();
+    StoreLists.clearChecked();
+    expect(JSON.parse(localStorage.getItem('todoList')).length).toEqual(0);
```

Comment:
- [ ] From what I understood, you cleared local storage, then you ran `clearChecked` function which won't clear anything since local storage is already cleared so we're not really testing if the function is running correctly. It's better if you follow the approach that the previous reviewer has mentioned 👍

Reply:
thanks for your suggestion, actually the function is working, i need to create a new object to test and local storage. clear() is to make sure everything is clear before I test, now fix that problem.Thanks for pointing out the right point.